docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md

```diff
@@ -126,3 +126,7 @@ The current implementation supports the assistant's responses only in text forma
 :::
 
 Ideally, the message's history must transit within the process and be the input of this `Converse` task with the new message.
+
+:::note
+Starting from version 8.7.0, the Amazon Bedrock Connector supports storing the response as a document. See additional details and limitations in [document handling](/components/concepts/document-handling.md).
+:::
```

Comment:
Not entirely correct: The Bedrock connector supports consuming documents as inputs for conversations, see [here](https://github.com/camunda/camunda-docs/blob/889b1f04f24b46eb1c59e7c2b3f215db55bba64b/docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md#converse)